index.html

```diff
@@ -29,7 +29,9 @@
 
     <main class="container">
       <div class="h1-row">
-        <h1>Entries</h1>
+        <h1 class="h1-entries">Entries</h1>
+        <h1 class="h1-new-entry hidden">New Entry</h1>
+        <h1 class="h1-edit-entry hidden">Edit Entry</h1>
```

Comment:
These shouldn't be 2 different elements. This should be 1 element, and we change that elements `textContent` depending on if they are editing / creating